csp/templatetags/csp.py

```diff
@@ -19,10 +19,9 @@ def __init__(self, nodelist):
 
     def render(self, context):
         output = self.nodelist.render(context)
-        nonce = context.get('request')
+        request = context.get('request')
+        nonce = request.csp_nonce() if hasattr(request, 'csp_nonce') else ''
```

Comment:
My 2 cents -- Is there a reason you're describing this as `request.csp_nonce()` over `request.csp_nonce`?